crates/bevy_input/src/gamepad.rs

```diff
@@ -1,12 +1,20 @@
 use crate::{Axis, Input};
 use bevy_app::{EventReader, EventWriter};
 use bevy_ecs::system::{Res, ResMut};
-use bevy_utils::HashMap;
+use bevy_utils::{tracing::info, HashMap, HashSet};
 
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
 #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
 pub struct Gamepad(pub usize);
 
+#[derive(Default)]
+/// Container of unique connected [Gamepad]s
+///
+/// [Gamepad]s are registered and deregistered in [gamepad_connection_system]
+pub struct GamepadLobby {
```

Comment:
I agree that GamepadLobby isn't the best name for this. Lets use the more descriptive `ActiveGamepads` unless someone can think of something better.

Reply:
Or maybe just `Gamepads`

Reply:
Yeah I like Gamepads more because it is terser

Reply:
Renamed it to `Gamepads`